configs/dyhead/README.md

```diff
@@ -0,0 +1,61 @@
+# Dynamic Head: Unifying Object Detection Heads with Attentions
+
+## Abstract
+
+<!-- [ABSTRACT] -->
+
+The complex nature of combining localization and classification in object detection has resulted in the flourished development of methods. Previous works tried to improve the performance in various object detection heads but failed to present a unified view. In this paper, we present a novel dynamic head framework to unify object detection heads with attentions. By coherently combining multiple self-attention mechanisms between feature levels for scale-awareness, among spatial locations for spatial-awareness, and within output channels for task-awareness, the proposed approach significantly improves the representation ability of object detection heads without any computational overhead. Further experiments demonstrate that the effectiveness and efficiency of the proposed dynamic head on the COCO benchmark. With a standard ResNeXt-101-DCN backbone, we largely improve the performance over popular object detectors and achieve a new state-of-the-art at 54.0 AP. Furthermore, with latest transformer backbone and extra data, we can push current best COCO result to a new record at 60.6 AP.
+
+<!-- [IMAGE] -->
+<div align=center>
+<img src="https://user-images.githubusercontent.com/42844407/149169448-fcafb6d0-b866-41cc-9422-94de9f1e1761.png" height="300"/>
+</div>
+
+<!-- [PAPER_TITLE: Dynamic Head: Unifying Object Detection Heads with Attentions] -->
+<!-- [PAPER_URL: https://arxiv.org/abs/2106.08322] -->
+
+## Citation
+
+<!-- [ALGORITHM] -->
+
+```latex
+@inproceedings{DyHead_CVPR2021,
+  author    = {Dai, Xiyang and Chen, Yinpeng and Xiao, Bin and Chen, Dongdong and Liu, Mengchen and Yuan, Lu and Zhang, Lei},
+  title     = {Dynamic Head: Unifying Object Detection Heads With Attentions},
+  booktitle = {Proceedings of the IEEE/CVF Conference on Computer Vision and Pattern Recognition (CVPR)},
+  year      = {2021}
+}
+```
+
+For results with the Swin Transformer backbones, you should cite the arXiv version, not the CVPR version.
+
+```latex
+@article{DyHead_arXiv2021,
```

Comment:
Usually, we can keep citing the CVPR version in such a case. The paper link can be linked to the arxiv version so that readers can find the newest paper.